Cargo.toml

```diff
@@ -18,7 +18,7 @@ wee_alloc = { version = "0.4", optional = true }
 
 [dev-dependencies]
 cfg-if = "1.0"
-chrono = "0.4"
+time = "0.3.7"
```

Comment:
Nit: There is no reason to pin to a specific version of this crate, use `0.3` instead.